linera-chain/src/certificate/confirmed.rs

```diff
@@ -20,11 +21,7 @@ impl GenericCertificate<ConfirmedBlock> {
     pub fn from_validated(validated: GenericCertificate<ValidatedBlock>) -> Self {
```

Comment:
I think this function can be removed: At its call site we don't actually need a certificate, just a value (to make a vote).